### DISCUSSION TITLE
How can I customize the toPlainText() method?

### DISCUSSION THREAD

**Discussion by maelchiotti** [singerdmx/flutter-quill#1665] [General]
Hi,

I am only using flutter-quill to allow the user to add checklists in the text field. I would like to customize the value of the `toPlainText()` method to insert a ✅ or a ⬜ in front of each lines if it is a checklist (and depending on its checked state).

So, this:
![image](https://github.com/singerdmx/flutter-quill/assets/44336112/58079a1a-a6b7-4b25-936a-d00d6807a065)

Would return this:
```
✅ first line
second line

⬜ fourth line
```

Instead of just the current return value:
```
first line
second line

fourth line
```

**Comment by EchoEllet** [comment]
You can create your own toPlainText, get the delta, and convert it to text to fit your needs
